package.json

```diff
@@ -19,40 +19,41 @@
     "build": "yarn run clean && tsc -p ./tsconfig.es2015.json && tsc -p ./tsconfig.cjs.json",
     "watch": "tsc -w -p ./tsconfig.es2015.json",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "deploy-storybook": "yarn storybook-to-ghpages --source-branch=main"
   },
   "keywords": [],
   "author": "Tiny Technologies",
-  "license": "Apache-2.0",
+  "license": "MIT",
   "dependencies": {
     "prop-types": "^15.6.2",
-    "tinymce": "^5.5.1"
+    "tinymce": "^6.0.0 || ^5.5.1"
```

Comment:
Work out how dependencies will work for Types of RawEditorOptions vs Settings

Reply:
Note for posterity: we have added this as a post-standup discussion since it effects a few integrations.

Reply:
I thought of this idea to get the type in a version agnostic way:
```ts
type EditorOptions = Parameters<TinyMCE['init']>[0];
```

Reply:
I have confirmed that this works on codesandbox. An easy way to check is to look at the type hints for `image_upload_handler` which changed between 5 and 6:
https://codesandbox.io/s/tinymce-react-forked-j4jjco?file=/src/index.js